dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.37/V2_37_26__Add_Fix_Columns_in_Visualization.sql

```diff
@@ -0,0 +1,9 @@
+-- This script relates to the task https://jira.dhis2.org/browse/DHIS2-11057
+
+-- Add new column "fixColumnHeaders"
+ALTER TABLE visualization ADD COLUMN IF NOT EXISTS fixColumnHeaders BOOLEAN;
+UPDATE visualization SET fixColumnHeaders = FALSE;
```

Comment:
Maybe add WHERE fixColumnheaders IS NULL and WHERE fixRowHeaders IS NULL?

Reply:
They are brand new boolean columns, so we want to set them to false in any case (being null or not).

Reply:
Since these are primitive booleans, we should make the column not-null as well, to avoid getting null values in the future, which will lead to a NPE (e.g. from direct SQL inserts).

ALTER TABLE visualization ALTER COLUMN fixColumnHeaders SET NOT NULL;

The alternative is to to set a default value of false for the column.

Reply:
Done